src/core/server/models/story/index.ts

```diff
@@ -61,6 +64,16 @@ export interface Story extends TenantResource {
    * createdAt is the date that the Story was added to the Talk database.
    */
   createdAt: Date;
+
+  /**
+   * moderation determines whether or not this is a PRE or POST moderated story.
+   */
+  moderation?: GQLMODERATION_MODE;
+
+  /**
+   * premodLinksEnable will put all comments that contain links into premod.
+   */
+  premodLinksEnable?: boolean;
```

Comment:
These settings should be available inside `settings.moderation` on the Story.

Reply:
I see what you mean!